modules/dataquery/jsx/react.filterBuilder.js

```diff
@@ -50,7 +50,9 @@ var FilterRule = React.createClass({
 				"lessThanEqual" : "<=",
 		    	"greaterThanEqual" : ">=",
 		    	"startsWith" : "startsWith",
-		    	"contains" : "contains"
+		    	"contains" : "contains",
+		    	"isNull" : "isNull",
+		    	"isNotNull" : "isNotNull"
```

Comment:
I can change the labels to any string, I'll discuss with Leigh and others what they think are the best labels.